terraform/main.tf

```diff
@@ -3,11 +3,22 @@ provider "aws" {
   region = var.aws_region
 }
 
+# This is the provider that can make changes to DNS entries
+# in the public cyber.dhs.gov zone.
+#
+# NOTE: After the CyHy AWS account moves in to the COOL, it will be possible
```

Comment:
in to --> into

Reply:
Thanks- see https://github.com/cisagov/cyhy_amis/pull/257/commits/6032cc87d1a934398e3964823253f067e81c30cc.